### ISSUE TITLE
[typedoc-plugin-markdown] Enum values are not generated

### ISSUE THREAD

**Issue by Zamiell** [tgreyuk/typedoc-plugin-markdown#323]
I think this might have to do with updates in the latest TypeDoc, but regardless, currently this plugin will delete all of the enum values.

For example, with the following source file:

```ts
export enum Foo {
  VALUE_1 = 0,
  VALUE_2 = 1,
}
```

This plugin produces the following Markdown:

```markdown
[test](../README.md) / [Exports](../modules.md) / Foo

# Enumeration: Foo

## Table of contents

### Enumeration Members

- [VALUE\_1](Foo.md#value_1)
- [VALUE\_2](Foo.md#value_2)

## Enumeration Members

### VALUE\_1

• **VALUE\_1**

#### Defined in

Foo.ts:2

___

### VALUE\_2

• **VALUE\_2**

#### Defined in

Foo.ts:3
```

In order words, it is missing the `= 0` and `= 1` parts.

**Comment by tgreyuk** [comment]
Thanks - Fix in typedoc-plugin-markdown@3.13.2

**Comment by Zamiell** [comment]
Thank you!!
